modular_event/arena_assets/bottomless_sign.dm

```diff
@@ -0,0 +1,16 @@
+/obj/structure/sign/bottomless
+	name = "bottomless pit"
+	desc = "I sure hope nobody puts a bottom in the pit."
+	icon_state = "bottomlesspit1"
+	icon = 'modular_event/arena_assets/sign2.dmi'
+	custom_materials = list(/datum/material/wood = 2000)
+
+/obj/structure/sign/bottomless/examine(mob/user)
+	. = ..()
+	. += span_nicegreen(">be me")
+	. += span_nicegreen(">bottomless pit supervisor")
+	. += span_nicegreen(">in charge of making sure the bottomless pit is, in fact, bottomless")
+	. += span_nicegreen(">occasionally have to go down there and check if the bottomless pit is still bottomless")
```

Comment:
I spoke with Mothblocks, they confirmed that you shouldn't have unescaped `>` in your text, can you see if it works with `&gt;`?